rc/filetype/zig.kak

```diff
@@ -102,8 +102,8 @@ define-command -hidden zig-trim-indent %{
 
 define-command -hidden zig-insert-on-new-line %<
     evaluate-commands -draft -itersel %<
-        # copy // or /// comments prefix and following whitespace
-        try %< execute-keys -draft k <a-x> s ^\h*\K///?\h* <ret> y gh j P >
+        # copy // or /// comments prefix or \\ string literal prefix and following whitespace
+        try %< execute-keys -draft k <a-x> s ^\h*\K(///?|\\\\)\h* <ret> y gh j P >
```

Comment:
im not sure copying following whitespace on multiline strings is wanted behaviour, but i don't think it matters that much